### DISCUSSION TITLE
Time Series Forecasting

### DISCUSSION THREAD

**Discussion by ChristoNindyo** [teamalgoritma/community#295] [Q&A] [answered]
Bagiamana cara memperbaiki x-axis seperti ini ? saya sudah lakuka ts()

![image](https://user-images.githubusercontent.com/105972495/189285624-c5770ad9-7c8a-43e1-8560-85022f64b666.png)

**Comment by NabiilahArdini** [comment, marked as answer]
Dear @ChristoNindyo , 
penentuan index `start` pada pembuatan objek ts harus disesuaikan dengan data. Pada data, Bapak memiliki rekam data harian, sehingga yang perlu dimasukan pada `start` contohnya seperti ini:

```
ts(data, 
   start = c(2010,01,01), # untuk year month date
   frequency = 365) # pola seasonal tahunan
```

Semoga dapat membantu!

**Reply by ChristoNindyo** [reply]
Terima kasih Mbak
